packages/blocks/src/api/constants.js

```diff
@@ -56,6 +56,10 @@ export const __EXPERIMENTAL_STYLE_PROPERTY = {
 		support: [ 'color', 'text' ],
 		requiresOptOut: true,
 	},
+	filter: {
+		value: [ 'filter', 'duotone' ],
+		support: [ 'color', '__experimentalDuotone' ],
```

Comment:
We don't really need declare `support`, do we? I mean, it's fine and even nice because it's whole. For a moment I thought whether this would make any block panel in the GS sidebar to show (despite not having any other support), but it seems we're fine there. Nothing to do here, just documenting my thought process and worries.

Reply:
This is actually the cause of the block instance issue https://github.com/WordPress/gutenberg/pull/37727#issuecomment-1085713264. It's adding the filter to the block container which we don't want.

Reply:
https://github.com/WordPress/gutenberg/pull/40185 is related although probably don't fully solve the issue (need to test more)

Reply:
40185 has landed and fixes this for 5.9. In 6.0 we need to get https://github.com/WordPress/wordpress-develop/pull/2542 in for the extra global stylesheet not being enqueued.